sycl/include/sycl/ext/intel/esimd.hpp

```diff
@@ -71,6 +71,14 @@
 
 /// @} sycl_esimd
 
+// TODO Disable "changes ABI" warnings emitted by host compiler, because ESIMD
```

Comment:
1. Can you given an example of the warning that you see?
2. Can't this mask potential problems with ABI differences when we use std library funcs, or something like that?

Reply:
1.
```
.../sycl/include/sycl/ext/intel/esimd/detail/simd_obj_impl.hpp:226:15: warning: AVX vector return of type 'cl::sycl::ext::intel::esimd::detail::simd_obj_impl<unsigned long, 16, cl::sycl::ext::intel::esimd::simd<unsigned long, 16>>::raw_vector_type' (aka 'cl::sycl::ext::intel::esimd::detail::raw_vector_type<unsigned long, 16>::type') without 'avx512f' enabled changes the ABI [-Wpsabi]
    set(other.data());
              ^
```
2. No. I tried to explain why in the comment.